packages/bits/src/lib/time-frame-picker/time-frame-picker.component.ts

```diff
@@ -38,6 +39,8 @@ export class TimeFramePickerComponent implements OnChanges, OnInit {
 
     public distanceToEndDate: number; // to keep distance between start and end-date
 
+    public onDestroy$ = new Subject<void>();
```

Comment:
Is this subject needed? I don't see where it's being used.

Reply:
will removed, thanks